gateway-provider-security-authc-remote/src/main/java/org/apache/knox/gateway/filter/RemoteAuthFilter.java

```diff
@@ -72,11 +72,14 @@ public class RemoteAuthFilter implements Filter {
   private static final String CONFIG_GROUP_HEADER = "remote.auth.group.header";
   private static final String DEFAULT_CONFIG_USER_HEADER = "X-Knox-Actor-ID";
   private static final String DEFAULT_CONFIG_GROUP_HEADER = "X-Knox-Actor-Groups-*";
+  private static final String CONFIG_TRUSTSTORE_PATH = "remote.auth.truststore.path";
```

Comment:
nit: the `remote.auth.` prefix could be a separate final constant (so that all properties could start like `REMOTE_AUTH_PREFIX + `)